package.json

```diff
@@ -13,6 +13,7 @@
     "@types/react-router-dom": "^5.1.7",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-player": "^2.8.2",
```

Comment:
I don't think we are using this so we can get rid of the changse to package.json and package-lock.json in this PR :)